protocol/messenger_communities.go

```diff
@@ -1724,16 +1719,17 @@ func (m *Messenger) CheckAndDeletePendingRequestToJoinCommunity(sendResponse boo
 			if notification != nil {
 				// Delete activity centre notification for community admin
 				if notification.Type == ActivityCenterNotificationTypeCommunityMembershipRequest {
-					notification.UpdatedAt = m.getCurrentTimeInMillis()
+					updatedAt := m.GetCurrentTimeInMillis()
+					notification.UpdatedAt = updatedAt
 					err = m.persistence.DeleteActivityCenterNotificationByID(types.FromHex(requestToJoin.ID.String()), notification.UpdatedAt)
 					if err != nil {
 						m.logger.Error("failed to delete notification from activity center", zap.Error(err))
 						return nil, err
 					}
 					notification.Deleted = true
-					err = m.syncActivityCenterNotifications([]*ActivityCenterNotification{notification})
+					err = m.syncActivityCenterDeletedByIDs(ctx, []types.HexBytes{notification.ID}, updatedAt)
```

Comment:
@qfrank I see this is not your code, it was already here, but I think this is wrong if I understand the context correctly. If a pending request to join a community gets deleted, we need to send that info and once it's received on paired devices will remove the pending request and delete the related notification from the activity center.

But, in my opinion, we don't need to delete a related activity center notification at all. Not sure why we're doing that. So we're adding a notification once a request is made, if it gets deleted in the meantime a click on that notification should open a popup saying "The request doesn't exist anymore" or something like that.

Reply:
not sure about this, this PR just want to implement what we've agreed on "we should be syncing decisions/states and not actual notifications" , what's your opinion? @ilmotta cc @cammellos